blocks/count-cards/count-cards.css

```diff
@@ -0,0 +1,45 @@
+.count-cards > ul {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+}
+
+.count-cards > ul > li {
+    border-left: 1px solid #dadada;
+    background-color: var(--background-color);
+    box-sizing: border-box;
+    color: rgb(4 20 89);
+    font-size: 16px;
+    font-weight: 400;
+    font-family: "Ringside Narrow", Arial, sans-serif;
```

Comment:
Not sure if we already set a font family for the entire website? Should we override this?

Reply:
I've removed this part.